egs/multi_en/s5/local/make_partitions.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2016  Allen Guo
+# Apache License 2.0
+
+multi=multi_a
+stage=1  # Notice that the $stage if-blocks use -eq in this script
+
+. utils/parse_options.sh
+
+pwd=`pwd`
+data_dir=data/$multi
+tmp1=$data_dir/tmp1
+tmp2=$data_dir/tmp2
+
+set -e
```

Comment:
Please exit the script with an error message. I would recommend `set -e` only in the top level script.